index.html

```diff
@@ -283,7 +283,7 @@ <h2 class="title-section">Proudly sponsored by</h2>
             For details regarding how to become a sponsor and the
             benefits for this event look at our
             <a id="sponsorship-prospectus" href="data/SponsorshipProspectus2023.pdf">
-              Sponsorship Prospectus
+              <strong style="color: blue">Sponsorship Prospectus </strong>
```

Comment:
@engagepy please prefer to use class instead of inline styling.

Reply:
It's only for this prospectus link. Hence avoided class. I'd like to retain it as is. What do you think ?

Reply:
It's better to avoid writing inline styles and we could use it later for more scenarios where we want some link to stand out. So a class can be added to styles.css for the same.

Reply:
Done